net/wol/Makefile

```diff
@@ -1,6 +1,6 @@
 PLUGIN_NAME=		wol
-PLUGIN_VERSION=		2.4
-PLUGIN_REVISION=	2
+PLUGIN_VERSION=		3.0
+PLUGIN_REVISION=	1
```

Comment:
```suggestion
PLUGIN_VERSION=		2.5
```

I think 2.5 should be exciting enough :)

Reply:
I've been wondering about this one. On one hand, the plugin change was not been big enough to justify big version bump, but on the other I wanted to indicate that this is different code base. 
Anyway I've changed the version according to your suggestion.